requirements.txt

```diff
@@ -18,10 +18,6 @@ click==6.7
 colorama==0.3.9
 coverage==4.5.1
 cryptography==2.8
-docker==3.4.0
-docker-compose==1.21.2
-docker-pycreds==0.3.0
-dockerpty==0.4.1
 docopt==0.6.2
```

Comment:
The above changes are irrelevant to the issue you are fixing. Please remove these changes from this PR.